lisa/util/shell.py

```diff
@@ -139,25 +140,52 @@ def generate_run_command(
 
 # retry strategy is the same as spurplus.connect_with_retries.
 @retry(Exception, tries=3, delay=1, logger=None)  # type: ignore
-def try_connect(connection_info: ConnectionInfo) -> Any:
+def try_connect(
+    connection_info: ConnectionInfo, ssh_timeout: int = 300, tries: int = 3
```

Comment:
Let's use 10 seconds as default value, and extend it in your test case.

Reply:
It looks the tries doesn't need now. Please remove it.

Reply:
tries is currently used for retrying `stdout.channel.recv_ready()`. From my understanding, this function seems to run a dummy command and check if we are getting the output. Do you think this is still required?

Reply:
I mean remove it from method arguments, but keep it internally. It looks it's not used. With this commit, this argument is confusing now. It's hard to explain the relationship between ssh_timeout and tries.